client.go

```diff
@@ -136,7 +195,10 @@ func (c *Client) RequestNoParse(r *Request) (*http.Response, error) {
 	// Fortunately, the initial response contains the cookie we need, so if we
 	// detect an empty response with cookies set that didn't have any errors,
 	// re-try the request while sending their cookies back to them.
-	if err == nil && response.ContentLength <= 0 && len(response.Cookies()) > 0 {
+	if err == nil &&
```

Comment:
This is where I'm talking about your discovercard fix interacting with a previous vanguard fix...

At the time I disliked the Vanguard fix and though it might be okay if it was the only one. I've discovered I was too optimistic. I expect discovercard will not be the last such hack, and I want to set the project up to be able to better handle these types of FI quirks in the future.